gsa/src/gmp/models/schedule.js

```diff
@@ -68,6 +68,29 @@ class Schedule extends Model {
 
     return ret;
   }
+
+  static fromObject(object) {
+    const ret = super.fromObject(object);
+
+    const {timezone, icalendar} = object;
+
+    if (hasValue(icalendar)) {
+      try {
+        ret.event = Event.fromIcal(icalendar, timezone);
+      } catch (error) {
+        log.error(
+          'Could not parse ical data of Schedule',
+          ret.id,
+          error,
+          icalendar,
+        );
+      }
+
+      // delete ret.icalendar;
```

Comment:
A comment on this. when i left this code in, I get errors saying I can't delete icalendar. Not sure why.